src/fastmcp/server/dependencies.py

```
@@ -784,6 +881,408 @@ async def introspect(server: FastMCP = CurrentFastMCP()) -> str:
+                await response_stream.aclose()
+                await response_stream_reader.aclose()
+
+            # Parse and return result
+            if result.action == "accept":
+                return handle_elicit_accept(config, result.content)
+            elif result.action == "decline":
+                return DeclinedElicitation()
+            elif result.action == "cancel":
+                return CancelledElicitation()
+            else:
+                raise ValueError(f"Unexpected elicitation action: {result.action}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)